ipa-core/src/query/runner/hybrid.rs

```diff
@@ -162,10 +176,8 @@ mod tests {
     const EXPECTED: &[u128] = &[0, 8, 5];
 
     fn build_records() -> Vec<TestRawDataRecord> {
```

Comment:
`TestRawDataRecord` is for the IpaOprfRerports, and we kept that around as we we're still decrypting `EncryptedOprfReport`s. We should be updating this to [`TestHybridRecords`](https://github.com/private-attribution/ipa/blob/main/ipa-core/src/test_fixture/hybrid.rs#L11). This will require a little bit of plumbing (I think you'll need to implement `IntoShares` to produce `HybridReport`s and `Reconstruct`to convert 3 `HybridReport`s back into a `TestHybridReport`.)